.tools_versions.yaml

```diff
@@ -1,7 +1,7 @@
 # renovate: datasource=github-tags depName=kubernetes/code-generator
 kube-code-generator: "0.32.1"
 # renovate: datasource=github-releases depName=kubernetes-sigs/controller-tools
-controller-tools: "0.17.1"
+controller-tools: "0.17.2-0.20250205103018-dac769d27bbe"
```

Comment:
This is needed to workaround broken aliases support in controller-gen https://github.com/kubernetes-sigs/controller-tools/issues/1136 (which wasn't released yet)